packages/cw0/src/expiration.rs

```diff
@@ -14,6 +15,16 @@ pub enum Expiration {
     Never {},
 }
 
+impl fmt::Display for Expiration {
```

Comment:
Useful for debugging

Reply:
Makes better error messages, right? sounds good.

Reply:
Exactly